### PR TITLE
feat: add deep archimedea language data

### PR DESCRIPTION
**Pull request by SlayerOrnstein**
### What did you fix? 
Added data for this weeks Deep Archimedea and corrected some missing or incorrectly implements test stuff for archonShard related functions

---

### Reproduction steps

---

### Considerations
- Does this contain a new dependency? **No**
- Does this introduce opinionated data formatting or manual data entry? **Yes**
- Does this pr include updated data files in a separate commit that can be reverted for a clean code-only pr? **Yes**
- Have I run the linter? **Yes**
- Is is a bug fix, feature request, or enhancement? **Feature**

### CODE REVIEW

File: test/utilities/translation.spec.js [hunk 1]
```diff
@@ -516,7 +516,7 @@ describe('translation', () => {
     });
     describe('archonShardColor()', () => {
       it("should return a translation of the key if it's found in the data", () => {
-        archonShardColor('ACC_BLUE', 'en').should.equal('Azure');
+        archonShardColor('ACC_BLUE', 'es').should.equal('Azure');
```
Comment (TobiTenno) [inline test/utilities/translation.spec.js:519]: is this just to ensure we're actually testing the translation? if so 👍

File: tools/translation.js [hunk 2]
```diff
@@ -335,6 +335,9 @@ export const translatePolarity = (pol = '') => valMapping(pol, polarityMap);
  * @property {function} toTitleCase      - Format provided string as titlecase
  * @property {function} translateFocus   - Translate focus schools
  * @property {function} translatePolarity - Translate polarities
+ * @property {function} archonShard      - Converts archon shard names
+ * @property {function} archonShardColor - Converts archon shard names to in-game color values
+ * @property {function} archonShardUpgradeType - Convert archon shard upgrade type
```
Comment (TobiTenno) [inline tools/translation.js:340]: excellent 👍

**Review by TobiTenno — APPROVED** [review]

File: test/utilities/translation.spec.js (hunk 1 above)
Comment (SlayerOrnstein) [reply test/utilities/translation.spec.js:519]: Yup, exactly that

File: tools/translation.js (hunk 2 above)
Comment (SlayerOrnstein) [reply tools/translation.js:340]: I'm evolving

**Pull request merged**